source/includes/options-mongosqld.yaml

```diff
@@ -671,4 +671,37 @@ args: <service-name>
 description: |
   Description of the system service which runs :binary:`~bin.mongosqld`.
 optional: true
+---
+program: mongosqld
+name: gssapiHostname
+directive: option
+args: <hostname>
+description: |
+  .. versionadded:: 2.5
+
+  Hostname that serves Kerberos.
+optional: true
+default: First IP address for :setting:`net.bindIp`.
+---
+program: mongosqld
+name: gssapiServiceName
+directive: option
+args: <service-name>
+description: |
+  .. versionadded:: 2.5
+
+  Name of the system service which runs the Kerberized :binary:`~bin.mongosqld`.
+optional: true
+default: mongosql
+---
+program: mongosqld
+name: mongo-gssapiServiceName
+directive: option
+args: <service-name>
+description: |
+  .. versionadded:: 2.5
+
+  Name of the system service which runs the Kerberized ``mongod``.
```

Comment:
this is equivalent to the mongo shell's --gssapiServiceName flag. Perhaps we can use the same documentation for this flag?